app/models/space.rb

```diff
@@ -49,5 +49,9 @@ def parent_location
     []
   end
 
+  def configured?(current_space)
+    current_space.rooms.blank? || current_space.entrance_id.blank? || current_space.members.blank? || current_space.entrance&.furnitures.blank?
+  end
```

Comment:
This might need refactoring later. I tried moving `configured?` to the Spaces controller but couldn't get it working. It's too late at night to keep messing with it. 

Maybe we can chat about it wednesday and fix whatever mental model got me stuck when I was working on this.

Reply:
Hmm, I wonder if what we want is a `Space::OnboardingComponent` that can have these methods? Then we could more easily test the whole kit and kaboodle, and we wouldn't be putting responsibility on the `Space` itself.